docs/types/literal-suffixes.rst

```diff
@@ -0,0 +1,243 @@
+
++-------------------------------------------------------------+----------------------------------------------------------------+
+| Parameter type                                              | Accepted literals                                              |
++=============================================================+================================================================+
+| ``bool``                                                    | - :ref:`Boolean<booleans>` literals                            |
++-------------------------------------------------------------+----------------------------------------------------------------+
+| ``uint8``, ..., ``uint256``, ``int8``, ..., ``int256``      | - :ref:`Rational literals<rational_literals>` (including zero) |
++-------------------------------------------------------------+----------------------------------------------------------------+
+| ``address``                                                 | - :ref:`Address literals<address_literals>`                    |
++-------------------------------------------------------------+----------------------------------------------------------------+
+| ``bytes1``, ..., ``bytes32``                                | - Hexadecimal number literals (not for ``bytes20``)            |
+|                                                             | - :ref:`Hexadecimal string literals<hexadecimal_literals>`     |
+|                                                             | - :ref:`String literals<string_literals>`                      |
+|                                                             | - :ref:`Unicode literals<unicode_literals>`                    |
+|                                                             | - :ref:`Zero<rational_literals>`                               |
++-------------------------------------------------------------+----------------------------------------------------------------+
+| ``bytes``                                                   | - :ref:`Hexadecimal string literals<hexadecimal_literals>`     |
+|                                                             | - :ref:`String literals<string_literals>`                      |
+|                                                             | - :ref:`Unicode literals<unicode_literals>`                    |
++-------------------------------------------------------------+----------------------------------------------------------------+
+| ``string``                                                  | - :ref:`String literals<string_literals>`                      |
+|                                                             | - :ref:`Unicode literals<unicode_literals>`                    |
++-------------------------------------------------------------+----------------------------------------------------------------+
+
+For two-parameter suffix functions, the first parameter (representing the mantissa) can be of any integer type.
```

Comment:
No, declaring `mantissa` as `int` will not let you get negative numbers in it unless you call it yourself using the function call syntax: `kg(-123, 2)`. The fact that allowing signed types here is misleading was actually one of the design questions we had. In the end we decided to allow it mostly because, if your result type is signed, you may avoid some superfluous explicit conversions this way or extra `require()`s to limit the range.

Reply:
I'll try to make it clearer.

Reply:
I mean to explicitly say for the two-parameter suffix function, that the `int` at the `mantissa` does not really mean it can deal with negative numbers.

Reply:
> So it could make sense to explicitly say in the documentation, that this will not work.

There is a note saying that just below this part.